src/resources/views/crud/fields/repeatable.blade.php

```diff
@@ -1,7 +1,9 @@
 {{-- REPEATABLE FIELD TYPE --}}
 
 @php
-  $field['value'] = old_empty_or_null($field['name'], []) ??  $field['value'] ?? $field['default'] ?? [];
+  $fieldNameForRepeatable = $field['real_name'] ?? $field['name'];
+  $field['value'] = old_empty_or_null($fieldNameForRepeatable, []) ??  $field['value'] ?? $field['default'] ?? [];
```

Comment:
If we did the above, this would still be:
```suggestion
  $field['value'] = old_empty_or_null($field['name'], []) ??  $field['value'] ?? $field['default'] ?? [];
```

But of course, it might have other implications, if `$field['name']` is used below for different purposes (and I assume it is).